src/architecture/system_model/sim_model.cpp

```diff
@@ -307,7 +335,7 @@ void SimModel::StepUntilStop(uint64_t SimStopTime, int64_t stopPri)
     }
     for(thrIt=this->threadList.begin(); thrIt != this->threadList.end(); thrIt++)
```

Comment:
I had to go back up and read the definition of the for loop to figure out what thrIt was, could be named something more clear like thread or curThread but this is a minor change and this name is not new in this PR so not a blocking issue.

Reply:
Good call and agreed. I change all these in the follow up PR https://github.com/lasp/basilisk/pull/210